openapi/anchor-platform/Callbacks API.yml

```diff
@@ -466,3 +466,34 @@ paths:
             application/json:
               schema:
                 $ref: './schemas.yml#/components/schemas/Error'
+  /events:
```

Comment:
I think we have singular names for the rest of our endpoints in this API right? I agree plural is the standard but we might want to stay consistent.

Reply:
Changed to singular.